bigtable/hello/main.py

```diff
@@ -70,7 +70,7 @@ def main(project_id, instance_id, table_id):
             row = table.row(row_key)
             row.set_cell(
                 column_family_id,
-                column_id.encode('utf-8'),
+                column_id,
```

Comment:
I think Jon is just fixing a double encode? I think in general if you want Python strings as bytes, do the encode, no six necessary.